services/booking-api/lambdas/create.js

```diff
@@ -0,0 +1,18 @@
+import { throwError } from '@helsingborg-stad/npm-api-error-handling';
+import to from 'await-to-js';
+
+import * as response from '../../../libs/response';
+
+import booking from '../helpers/booking';
+
+export async function main(event) {
+  const body = { ...JSON.parse(event.body) };
```

Comment:
Curious about why you are spreading here?

Reply:
Shouldn't be needed. Nice catch. Will remove it.

Reply:
The spreading is due to that the event.body is a JSON payload so that's fine. But i wounder what the JSON data represents ie what does the body contains, is it a new booking object?
Then i would suggest something like the following:
```JS
const bookingEventtBody = { ...JSON.parse(event.body) }
```